src/Compilers/VisualBasic/Portable/Binding/LookupResult.vb

```diff
@@ -691,6 +691,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                     CompactSymbols(lost)
                     Debug.Assert(_symList.Count > 0)
 
+                    ' As a special case, we allow conflicting enum members imported from
+                    ' metadata If the have the same value, and take the first
```

Comment:
Could you please adjust XML doc comments for the relevant APIs to make this contract (determinism) explicit and double check that all implementations of the APIs within compilers fulfill the contract?